src/game.cpp

```diff
@@ -175,6 +176,34 @@ bool is_valid_in_w_terrain(int x, int y)
     return x >= 0 && x < TERRAIN_WINDOW_WIDTH && y >= 0 && y < TERRAIN_WINDOW_HEIGHT;
 }
 
+namespace {
+
+using namespace std::chrono;
+
+time_point<steady_clock> user_turn_start;
```

Comment:
Since this is only active inside the player input method, the variable should live there, I'm looking at merging, so I'll make a follow-up PR that does this.